daphne/src/constants.rs

```diff
@@ -8,6 +8,7 @@
 // TODO spec: Decide if media type should be enforced. (We currently don't.) In any case, it may be
 // useful to enforce this for testing purposes.
 pub const MEDIA_TYPE_HPKE_CONFIG: &str = "application/dap-hpke-config";
```

Comment:
Suggestion: To make it clear that this media type is only kept for backwards compatibility, rename it accordingly.
```suggestion
pub const DRAFT_02_MEDIA_TYPE_HPKE_CONFIG: &str = "application/dap-hpke-config";
```